roles/etcd/templates/etcd.conf.j2

```diff
@@ -11,18 +23,20 @@
 # http_read_timeout = 10
 # http_write_timeout = 10
 # key_file = ""
-# peers = []
+{% if etcd_cluster %}
+peers = [ {{ etcd_peers() }}]
+{% endif %}
 # peers_file = ""
 # max_cluster_size = 9
 # max_result_buffer = 1024
 # max_retry_attempts = 3
-# name = "default-name"
+name = "{{ ansible_fqdn }}"
```

Comment:
Does this name have to be the hostname, or just a unique name for the cluster? We could always generate a unique name instead of relying on the hostname.